storey/table.py

```diff
@@ -24,7 +24,7 @@ class Table:
      """
 
     def __init__(self, table_path: str, storage: Driver, partitioned_by_key: bool = True,
-                 flush_interval_secs: Optional[int] = 300, max_updates_in_flight: int = 8):
+                 flush_interval_secs: Optional[int] = 300, max_updates_in_flight: int = 8, support_aggr=True):
```

Comment:
What happens if `support_aggr=False` in ingest rather than on querying? Looks like this parameter only has one specific flow in mind...